### DISCUSSION TITLE
EEPROM.BIN backups, 2nd TX

### DISCUSSION THREAD

**Discussion by lewis6x** [OpenI6X/opentx#414] [General]
Just when I thought I understood this...

I figured I could copy the EEPROM.BIN from one FS-i6X with openi6x-1.11.1 to another one and it would operate the configured models. While the models show up they are not operable unless I bind them to the 2nd TX and at that point they don't operate from the original TX. 

Is this normal?  If so I'm guessing there is a token imbedded in each TX that only allows binding to one TX. Is there a way around this? It would be handy to leave a TX at the shop and not have to rebind to the TX that stays at the lake.

**Comment by ajjjjjjjj** [comment]
It is expected behaviour - TX have unique ID.
It can be solved only by doing changes in source code to use predefined ID.

**Comment by lewis6x** [comment]
Thanks! I'm starting to "get" it!

**Comment by Snekovo** [comment]
it would be great if you could copy the model from eeprom.bin and paste it into another transmitter, not only 1:1 backup :)
